guides/source/configuring.md

```diff
@@ -1131,51 +1131,13 @@ config.relative_url_root = "/app1"
 alternatively you can set the `RAILS_RELATIVE_URL_ROOT` environment
 variable.
 
-Rails will now prepend "/app1" when generating links.
```

Comment:
I'm confused about this too. In my testing, I see no change to generated URLs. I'll investigate further to see what the code is doing.

Reply:
Hmm, my experiment with a fresh Rails app gives a different picture. I put `config.relative_url_root = "/abc"` into `/config/application.rb`
The effect is:
- asset paths are prefixed by `/abc`  (as you would expect)
- url helpers in views do **not** generate prefix `/abc`
- url helpers in integration tests do generate prefix `/abc`; thus, tests fail

Sample application is here: https://github.com/krusty3002/rails-pr-34539

Reply:
🤔 You're right. I was testing against the output of `Rails.application.routes.url_helpers` but views and controller seem to still be generating the wrong (unprefixed) paths... I'll keep digging.